api/api/projects_api.go

```diff
@@ -78,13 +78,19 @@ func (c *ProjectsController) UpdateProject(r *http.Request, vars map[string]stri
 	project.Team = newProject.Team
 	project.Stream = newProject.Stream
 	project.Labels = newProject.Labels
-	project, err = c.ProjectsService.UpdateProject(r.Context(), project)
+	updatedProject, responseMessage, err := c.ProjectsService.UpdateProject(r.Context(), project)
 	if err != nil {
 		log.Errorf("error updating project %s: %s", project.Name, err)
 		return FromError(err)
 	}
 
-	return Ok(project)
+	if responseMessage != "" {
+		log.Infof("Project updated successfully, URL link: %s", responseMessage)
+	} else {
+		log.Infof("Project updated successfully")
+	}
+
+	return Ok(updatedProject)
```

Comment:
the url link will not be passed to the api response kinda make new floor pointless, this is not intended right?